src/containers/reports/ReportResult.js

```diff
@@ -22,6 +22,17 @@ import { REPORT_VIEW_PATH, REPORT_ID_PATH } from '../../core/router/Routes';
 const { OPENLATTICE_ID_FQN } = Constants;
 const { NEUTRALS } = Colors;
 
+const StyledCard = styled(Card)`
+  :hover {
+    box-shadow: rgba(0, 0, 0, 0.07) 0px 5px 15px 0px;
+    cursor: pointer;
```

Comment:
Thoughts on adding these styles to the stock Card (LUK) component whenever an onClick is present?

Reply:
I'll adjust the styling moving forward. 

I feel detecting onClick is the most obvious place to put it. I can see it being undesirable if someone wanted this style to not be applied, but I feel if a Card does have an onClick, there definitely should be some indication of a click action. The style can ofc be adjusted through `styled(Card)` but I think in the end, SOME style should be applied whenever onClick is present.